src/phrases/phrases-nb.ts

```diff
@@ -120,6 +120,8 @@ const phrases = {
     lastUsed: 'Sist redigert',
     savedSearch: 'Lagrede søk',
     saveSearch: 'Lagre søk',
+    welcomeBack: 'Velkommen tilbake',
+    addEmptyArticle: 'Opprett tom artikkel',
```

Comment:
Alle nye artikler er tomme, så kanskje det hadde vore bedre å bruke "Opprett læringsressurs" som er det samme som brukes i menyen i dag? For dette valget gjør jo kun det enn så lenge.